AirWallex/Airwallex/Sources/AWXPaymentMethodResponse.m

```diff
@@ -66,7 +66,62 @@ @implementation AWXGetPaymentMethodTypesResponse
 
 @end
 
+@interface AWXGetPaymentMethodTypeResponse ()
 
+@end
+
+@implementation AWXGetPaymentMethodTypeResponse
+
++ (id<AWXResponseProtocol>)parse:(NSData *)data
+{
+    NSError *error = nil;
+    id responseObject = [NSJSONSerialization JSONObjectWithData:data options:NSJSONReadingMutableContainers error:&error];
+    AWXGetPaymentMethodTypeResponse *response = [AWXGetPaymentMethodTypeResponse new];
+    response.name = responseObject[@"name"];
+    response.displayName = responseObject[@"display_name"];
+    response.logoURL = responseObject[@"logo_url"];
```

Comment:
the structure of the response is:
```
{
    "name": "bank_transfer",
    "display_name": "Bank transfer",
    "logos": {
        "png": "https://checkout-staging.airwallex.com/static/media/bank_transfer.888333.png",
        "svg": "https://checkout-staging.airwallex.com/static/media/bank_transfer.0a61bea6.svg"
    },
...
}
```
you can use `logos.png`. `logo_url` will not be returned.